pkg/kapp/diffgraph/change_graph_test.go

```diff
@@ -809,6 +810,34 @@ metadata:
 	require.Equal(t, expectedOutput, output)
 }
 
+func TestChangeGraphWithSecretAndSA(t *testing.T) {
+	yaml := `
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: sa-0
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: secret-0
+  annotations:
+    kubernetes.io/service-account.name: sa-0
+type: kubernetes.io/service-account-token
+`
+
+	graph, err := buildChangeGraph(yaml, ctldgraph.ActualChangeOpUpsert, t)
+	require.NoErrorf(t, err, "Expected graph to build")
+
+	output := strings.TrimSpace(graph.PrintStr())
+	fmt.Println(output)
+	expectedOutput := strings.TrimSpace(`
+(upsert) serviceaccount/sa-0 (v1) cluster
+(upsert) secret/secret-0 (v1) cluster
```

Comment:
Shouldn't there be a tab here to indicate that the secret is ordered after service account? 🤔 (I haven't looked at these tests in a while so not sure)

Reply:
I think the `strings.TrimSpace` usage would strip the tab anyways

Reply:
Yes, strings.TrimSpace strip the tab as well.